Fixes:

1.  Dependency 'com.termux:termux-shared:0.118.0' requires desugar_jdk_libs version to be
           2.1.2 or above for :app, which is currently 1.1.5

--- a/termux-api/app/build.gradle
+++ b/termux-api/app/build.gradle
@@ -66,7 +66,7 @@ android {
 }
 
 dependencies {
-    coreLibraryDesugaring "com.android.tools:desugar_jdk_libs:1.1.5"
+    coreLibraryDesugaring "com.android.tools:desugar_jdk_libs:2.1.2"
 
     implementation "com.google.android.material:material:1.12.0"
     implementation "androidx.biometric:biometric:1.2.0-alpha05"
--- a/termux-float/app/build.gradle
+++ b/termux-float/app/build.gradle
@@ -66,7 +66,7 @@ android {
 }
 
 dependencies {
-    coreLibraryDesugaring "com.android.tools:desugar_jdk_libs:1.1.5"
+    coreLibraryDesugaring "com.android.tools:desugar_jdk_libs:2.1.2"
 
     testImplementation "junit:junit:4.13.2"
 
--- a/termux-tasker/app/build.gradle
+++ b/termux-tasker/app/build.gradle
@@ -67,7 +67,7 @@ android {
 }
 
 dependencies {
-    coreLibraryDesugaring "com.android.tools:desugar_jdk_libs:1.1.5"
+    coreLibraryDesugaring "com.android.tools:desugar_jdk_libs:2.1.2"
 
     testImplementation "junit:junit:4.13.2"
     androidTestImplementation "androidx.test.ext:junit:1.2.1"
--- a/termux-widget/app/build.gradle
+++ b/termux-widget/app/build.gradle
@@ -45,6 +45,8 @@ android {
     }
 
     compileOptions {
+        coreLibraryDesugaringEnabled true
+
         sourceCompatibility JavaVersion.VERSION_11
         targetCompatibility JavaVersion.VERSION_11
     }
@@ -63,6 +65,7 @@ android {
 }
 
 dependencies {
+    coreLibraryDesugaring "com.android.tools:desugar_jdk_libs:2.1.2"
     // required for TermuxWidgetControlsProviderService
     implementation "org.reactivestreams:reactive-streams:1.0.3"
     implementation "io.reactivex.rxjava2:rxjava:2.2.10"
